frontend/src/app/main/ui/ds/controls/switcher/switcher.mdx

```diff
@@ -0,0 +1,161 @@
+{ /* This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+  Copyright (c) KALEIDOS INC */ }
+
+import { Canvas, Meta } from '@storybook/blocks';
+import * as SwitcherStories from "./switcher.stories";
+
+<Meta title="DS/Controls/Switcher" />
+
+# Switcher
+
+The `switcher*` component is a toggle control that allows users to switch between two states (on/off). It provides both controlled and uncontrolled modes, making it suitable for various use cases across the interface.
+
+<Canvas of={SwitcherStories.DefaultUncontrolled} />
+
+## Anatomy
+
+The switcher component consists of three main parts:
+
+- **Label** (optional): Text that describes what the switcher controls
+- **Track**: The pill-shaped background that indicates the current state
+- **Thumb**: The circular knob that moves between positions
+
+## Props
+
+| Name | Type | Default | Description | Required |
+|------|------|---------|-------------|----------|
+| `id` | `string` | auto-generated | Unique identifier for the switcher | No |
+| `label` | `string` | - | Text label displayed next to the switcher | No |
+| `checked` | `boolean` | - | Controlled checked state | No |
+| `default-checked` | `boolean` | `false` | Initial checked state for uncontrolled mode | No |
+| `on-change` | `function` | - | Callback fired when state changes `(new-checked, event) => void` | No |
+| `disabled` | `boolean` | `false` | Whether the switcher is disabled | No |
+| `size` | `"sm" \| "md" \| "lg"` | `"md"` | Size variant of the switcher | No |
+| `aria-label` | `string` | - | Accessible label when no visible label is provided | No |
+| `class` | `string` | - | Additional CSS classes | No |
+| `data-testid` | `string` | - | Test identifier for automated testing | No |
+
+## Usage Examples
+
+### Controlled vs Uncontrolled
+
+#### Uncontrolled (manages its own state)
+```clj
+[:> switcher* {:label "Enable notifications"
+               :default-checked false
+               :on-change (fn [checked event] 
+                           (js/console.log "New state:" checked))}]
+```
+
+#### Controlled (state managed externally)
+```clj
+(let [checked (r/atom false)]
+  [:> switcher* {:label "Enable notifications"
+                 :checked @checked
+                 :on-change (fn [new-checked event]
+                             (reset! checked new-checked))}])
+```
+
+<Canvas of={SwitcherStories.Controlled} />
+
+### Different Sizes
+
+The switcher supports three size variants: `"sm"`, `"md"` (default), and `"lg"`.
+
+```clj
+[:> switcher* {:size "sm" :label "Small"}]
+[:> switcher* {:size "md" :label "Medium"}]
+[:> switcher* {:size "lg" :label "Large"}]
+```
+
+<Canvas of={SwitcherStories.Sizes} />
+
+### Disabled State
```

Comment:
Yo can set it as a control on the story, no need to add a complete different story for this.